### PR TITLE
Adding request module(check url existance), modularizing posts...

### PR DESCRIPTION
**Pull request by marvelousdesign**
…updating readme.

Using request module to check for a url's existence, but it also return undefined on websites that don't match their certificate (bad practice in general?).

Modularizing posts.js and postVal.js.
Index.js will call for postVal.js.
postsVal.js chains to posts.js, it checks for valid urls then saves..

Updated readme with the example markup style guideline.

### CODE REVIEW

File: client/js/createPost.js [hunk 1]
```diff
@@ -21,9 +19,22 @@ const createPost = () => {
     }).then((response) => {
         return response.json()
     }).then((data) => {
-        console.log('Message:', data.message)
+        // url2 output what process.env.VARIABLE is between BASE_URI & LOCALHOST in server/routes/db.js
+        const getServerUrl = (data) => {
+            return (data.baseUrl === 'https://gmshortener.herokuapp.com/') ?
```
Comment (ayyrickay) [inline client/js/createPost.js]: is there a reason you can't just return the baseUrl?

Comment (ayyrickay) [reply client/js/createPost.js]: It seems like baseUrl should be localhost on local, and gmshortener on prod, and both point to the same process.env

Comment (marvelousdesign) [reply client/js/createPost.js]: Ahh, I misunderstood somewhere, but will fix it in the next commit.

Comment (ayyrickay) [reply client/js/createPost.js]: No problem. (: Just open a new issue and close the main one for deploying to heroku, this isn't a bad problem to have

**Pull request merged**